packages/utils/src/dal/mikro-orm/mikro-orm-create-connection.ts

```diff
@@ -33,7 +34,7 @@ export async function mikroOrmCreateConnection(
     tsNode: process.env.APP_ENV === "development",
     type: "postgresql",
     migrations: {
-      path: __dirname + "/../migrations",
+      path: pathToMigrations,
```

Comment:
question: It looks like we are passing `__dirname + "/../migrations"` all places. What caused the need for this change?

Reply:
`__dirname` will take the path of the calling function. When done within the helper, it'll be `node_modules/@medusajs/utils/dal/mikro-orm/../migrations`, where it should have been `node_modules/@medusajs/product/loaders/../migrations`

Reply:
Ahh got it! Didn't think of that, thx!

Reply:
Running `__dirname +  "/../migrations"` in the DAL-util tries to access the migrations folder of the util rather than the module and will cause migrations to fail

Reply:
beat me to it 😂